src/sentry/integrations/utils/code_mapping.py

```diff
@@ -73,23 +71,35 @@ def stacktrace_buckets(self, stacktraces: List[str]) -> Dict[str, Any]:
                 continue
         return buckets
 
-    def generate_code_mappings(self, stacktraces: List[str]) -> List[CodeMapping]:
```

Comment:
If you make your browser window wide and use the Split view, the diff will be easier to view:
<img width="173" alt="image" src="https://user-images.githubusercontent.com/44410/197618550-070b29e5-fc16-47e5-a2bd-b410fa9425d9.png">